r/tests/testthat/test-dplyr-string-functions.R

```diff
@@ -20,6 +20,7 @@ skip_if_not_available("utf8proc")
 
 library(dplyr)
 library(stringr)
+library(stringi)
```

Comment:
Without stringi in the suggests, loading it causes an R CMD check warning.

Since I think it's best to avoid changing the suggests unless it's really necessary, I think the best solution is:
- Remove this `library(stringi)`
- Rewrite the tests of `stri_reverse` to use `expect_equal()` instead of `expect_dplyr_equal()`, and use `stringi::stri_reverse` in the dplyr in the second argument to it. So for example the first test would become:
    ```r
    expect_equal(
     df_utf8 %>%
       Table$create() %>%
       mutate(x = stri_reverse(x)) %>%
       collect(),
     df_utf8 %>%
       mutate(x = stringi::stri_reverse(x))
   )
   ```